### ISSUE TITLE
Enable use of Guacamole extensions

### ISSUE THREAD

**Issue by bkraegelin** [boschkundendienst/guacamole-docker-compose#50]
According to https://guacamole.apache.org/doc/gug/guacamole-docker.html#custom-extensions-and-guacamole-home you are able to install and use guacamole extensions.

Could you provide a _standard way_ within your docker-compose to install extensions?
A default directory to mount and a GUACAMOLE_HOME environment variable to point to it?

Just to be safe for any updates.

Thanks,
Birger

**Comment by boschkundendienst** [comment]
I can also only point you to this page: https://guacamole.apache.org/doc/gug/guacamole-docker.html#custom-extensions-and-guacamole-home

Especially this section:

**Because the Docker image’s GUACAMOLE_HOME environment variable must point to a directory within the container, you will need to expose your custom GUACAMOLE_HOME to the container using the -v option of docker run. The container directory chosen can then be referenced in the GUACAMOLE_HOME environment variable, and the image will handle the rest automatically:**

So basically you have to expose the GUACAMOLE_HOME directory somehow modifying the docker-compose.yml file and then follow the instructions.
